docs/OV_Runtime_UG/supported_plugins/AutoPlugin_Debugging.md

```diff
@@ -0,0 +1,137 @@
+# Debugging Auto-Device Plugin {#openvino_docs_IE_DG_supported_plugins_AUTO_debugging}
+
+## Debug log
+In case of execution problems, just like all other plugins, Auto-Device provides the user with information on exceptions and error values. If the returned data is not enough for debugging purposes, more information may be acquired by means of ov::log::Level.
+
+There are six levels of logs, which can be called explicitly or set via the OPENVINO_LOG_LEVEL environment variable (can be overwritten by compile_model or set_property):
+
+0 - ov::log::Level::NO  
+1 - ov::log::Level::ERR  
+2 - ov::log::Level::WARING  
+3 - ov::log::Level::INFO  
+4 - ov::log::Level::DEBUG  
+5 - ov::log::Level::TRACE  
+
+@sphinxdirective
+.. tab:: C++ API
+
+   .. code-block:: cpp
+
+      ov::Core core;
+
+      // read a network in IR, PaddlePaddle, or ONNX format
+      std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
+
+      // load a network to AUTO and set log level to debug
+      ov::CompiledModel compiled_model = core.compile_model(model, "AUTO", {{ov::log::level.name(), "LOG_DEBUG"}});
```

Comment:
1. Use model instead of network
2. Use proper way to set property
3. set_config and load network => set_property and compile_model